src/index.ts

```diff
@@ -108,6 +109,8 @@ export {
   BBox,
   LocationIdSHv3,
   setDebugEnabled,
+  CancelFactory,
+  isCancelled,
```

Comment:
We are probably missing `RequestConfiguration` export here?

Reply:
True, thanks!